index.html

```diff
@@ -162,25 +162,39 @@ <h2 class="salutation">Tonic</h2>
             <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
         </ul> 
     </section>  
+    <div>
+        <button class="get-resume" type="button">Get my Resume</button>
 
-            
-</div>
-<button class="get-resume" type="button">Get my Resume</button>
-</div>
-<section class="skills">
-
-<ul class="about-skill">
-    <li class="aboutl">Languages <img src="images/arrowDown.jpg" alt="">
-    </li> 
-    <p><img src="images/js.jpg" alt="">JavaScript</p>
-    <p><img src="images/html.jpg" alt="">HTML</p>
-    <p><img src="images/ccs3.jpg" alt="">CSS</p>
+    </div>
     
-    <li class="about">Frameworks <img src="images/arrowRight.jpg" alt=""></li> 
-    <hr>
-    <li class="about">Skills <img src="images/arrowRight.jpg" alt=""></li> 
-    <hr>
-</ul>
+    <section class="skills">
 
-    
-</section>
\ No newline at end of file
+        <ul class="about-skill">
+            <li class="aboutl">Languages <img src="images/arrowDown.jpg" alt=""></li> 
+            <p><img src="images/js.jpg" alt="">JavaScript</p>
+            <p><img src="images/html.jpg" alt="">HTML</p>
+            <p><img src="images/ccs3.jpg" alt="">CSS</p>
+            
+            <li class="about">Frameworks <img src="images/arrowRight.jpg" alt=""></li> 
+            <hr>
+            <li class="about">Skills <img src="images/arrowRight.jpg" alt=""></li> 
+            <hr>
+        </ul>
+    </section>
+</section>
+
+<!-- This section is about contact form -->
+<section class="contact-form">
+    <h2 class="contact-me ">Contact me</h2>
+    <p class="contact-item">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+        <form action="#" class="form contact-item">
+            <input class="form-item" type="text" maxlength="30" placeholder="Lorem Ipsum" required>
+            <input class="form-item" type="email" placeholder="loremipsum@gmail.com" required>
+            <textarea class="form-item" name="comment" id="comment" cols="50" rows="10" placeholder="Write your message here..."></textarea>
```

Comment:
- [ ] Please make sure that all input fields have the `required` attribute to activate the validation on submit as given in the project requirements ⛏🤓
- [ ] Also kindly ensure that the message input accepts only up to `500 chars`. You can use `maxlength` attribute for it ✨

  > _Message field accepts up to 500 chars._
  > _A user cannot submit the form if one or more of the fields is empty._